10-dhcp-config.conf

```diff
@@ -1,2 +0,0 @@
-[main]
```

Comment:
Please format the commit message. The title line should be under 70 characters. The rest should be divided in a paragraph bellow. You can see it here that even GitHub gets confused by the long line and it does not help readability of the log.

You say that these files just get created. Is there any guidance on how to create them? Is it just a regular process while working with RHCOS and we assume folks know how to generate it?